docker-compose.yml

```diff
@@ -1307,6 +1307,8 @@ services:
       ARROW_HDFS_TEST_HOST: impala
       ARROW_HDFS_TEST_PORT: 8020
       ARROW_HDFS_TEST_USER: hdfs
+      ARROW_S3: "OFF"
```

Comment:
Also, the answer is probably "yes", but I just wanted to make sure you meant to include this?

Reply:
Yes, it makes that build a little faster.